src/Layer/PointCloudLayer.js

```diff
@@ -0,0 +1,595 @@
+import * as THREE from 'three';
+import GeometryLayer from 'Layer/GeometryLayer';
+import Fetcher from 'Provider/Fetcher';
+import PotreeBinParser from 'Parser/PotreeBinParser';
+import PotreeCinParser from 'Parser/PotreeCinParser';
+import PointsMaterial, { MODE } from 'Renderer/PointsMaterial';
+import Picking from 'Core/Picking';
+import Extent from 'Core/Geographic/Extent';
+import CancelledCommandException from 'Core/Scheduler/CancelledCommandException';
+
+const point = new THREE.Vector3();
+// Draw a cube with lines (12 lines).
+function cube(size) {
```

Comment:
This could be replaced with `THREE.BoxGeometry` ?

Reply:
yes, in next PR

Reply:
yes, may be with OBB Helper